routes/orders.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const ordersRouter = express.Router();
+const { requireUser, isAdmin } = require("./utils");
+
+ordersRouter.get("/orders", [requireUser, isAdmin], async (req, res, next) => {
```

Comment:
Should just change to `/` instead of `/orders`

Same with the rest of these routes.